Lib/multiprocessing/shared_memory.py

```diff
@@ -493,18 +614,22 @@ def __init__(self, name, segment_names=[]):
         self.shared_memory_context_name = name
         self.segment_names = segment_names
 
-    def register_segment(self, segment):
-        util.debug(f"Registering segment {segment.name!r} in pid {os.getpid()}")
-        self.segment_names.append(segment.name)
+    def register_segment(self, segment_name):
```

Comment:
This is a public name but it's not documented. Shouldn't it be private?

Reply:
I have debated whether `SharedMemoryTracker` should be made private or properly documented as public.

In my own use of this code over the last year, I have found reasons to use `SharedMemoryTracker` independent of its use through `SharedMemoryManager`.  But I am unconvinced that it will find general use and that my use of it is only an edge case.

I hope that someone will react to it during review and say, "this is awesome, why is this not documented properly?!?"  If no one has a strong reaction, then I agree it should be private to the module.

Reply:
Changed ``SharedMemoryTracker`` to be ``_SharedMemoryTracker`` in commit 594140a65489adde4f60b14925e295bb1639501e.